opentelekomcloud/acceptance/dns/resource_opentelekomcloud_dns_recordset_v2_test.go

```diff
@@ -158,6 +158,38 @@ func TestAccDNSV2RecordSet_shared(t *testing.T) {
 	})
 }
 
+func TestAccDNSV2RecordSet_txt(t *testing.T) {
+	var recordset recordsets.RecordSet
+	zoneName := randomZoneName()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { common.TestAccPreCheck(t) },
+		ProviderFactories: common.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckDNSV2RecordSetDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDNSV2RecordSetTxt(zoneName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDNSV2RecordSetExists(resourceRecordSetName, &recordset),
+					resource.TestCheckResourceAttr(resourceRecordSetName, "name", zoneName),
+					resource.TestCheckResourceAttr(resourceRecordSetName, "description", "a record set"),
+					resource.TestCheckResourceAttr(resourceRecordSetName, "type", "TXT"),
+					resource.TestCheckResourceAttr(resourceRecordSetName, "ttl", "300"),
+					resource.TestCheckResourceAttr(resourceRecordSetName, "records.0", "v=spf1 include:spf.protection.outlook.com -all"),
```

Comment:
why it is here?

Reply:
for checks that string properly saved in state

Reply:
spf.protection.outlook.com - this url, i mean.

Reply:
changed for example